en/05_Contributing/01_Code.md

```diff
@@ -113,9 +111,16 @@ Other entities might be considered to be included or excluded from the public AP
 
 Any updates to third party dependencies in composer.json should aim to target the default branch for a minor release if possible. Targeting a patch release branch is acceptable if updating dependencies is required to fix a bug.
 
+API from third party dependencies may indirectly be incorporated into Silverstripe CMS's API if:
+- they are define as a parameter type for a supported method
+- they are define as a return type for a supported method
+- they are extended by a Silverstripe CMS class.
```

Comment:
What about if they are provided as an explicitly injectable service?

Reply:
"explicitly injectable service" didn't get brought up in the RFC. It probably should be discussed in a follow up discussion.

Reply:
https://github.com/silverstripe/silverstripe-framework/issues/10345#issuecomment-1155769208
> I think that it should include considerations around the fact that a developer could access (say) `protected` methods and properties through `Injector`.

It was briefly mentioned, though using different wording and didn't really get any traction as a talking point. I'm okay with it being discussed in a follow-up discussion so long as it does _actually_ get discussed, because IMO if we're setting up an `Injectable` service people should be able to rely on that service existing and being of a consistent type.

Reply:
I can't remember the last time a PR came up where this would have been an issue. It's mostly an academic point. Resolving this ambiguity is a low priority from my end. 

If you want to create a follow up PR to clarify when an Injectable object should be part of our API, I'm happy to have a look at it.

Reply:
A follow-up PR would likely be pointless at this stage - I already made a recommendation (albeit in the format of a question) and it was effectively rejected at this stage, with the suggestion of a follow-up discussion. It sounds like discussion is the next step. That needn't hold up this PR, so I'm fine with that discussion happening separately from this PR after this is merged.